RocketChat/Rocket.Chat
# fix: moment locale change

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

 - Change language to some other language (Let’s say English to Arabic)
 - Check the timestamps in chat, should be available in new selected language (timestamps in Arabic)
 - Now change the language back to previous language (Change to English from Arabic)
 - The timestamps are still shown in old language (timestamps are shown in Arabic instead of English)
## Further comments

TC-944
